src/Gui/DlgDisplayPropertiesImp.cpp

```diff
@@ -184,6 +187,13 @@ void DlgDisplayPropertiesImp::slotChangedObject(const Gui::ViewProvider& obj,
                                                        (int)(255.0f*value.b)));
                 d->ui.buttonLineColor->blockSignals(blocked);
             }
+            else if (prop_name == "PointColor") {
+                bool blocked = d->ui.buttonPointColor->blockSignals(true);
+                d->ui.buttonPointColor->setColor(QColor((int)(255.0f*value.r),

```

Comment:
all of this is copy paste adapted to point color instead of lineColor, most of this is actually triplicated since the line color code also seems just a copy paste from shape color, I don't understand enough to unify everything though so I just did copy/paste. Do you want me then to put the spaces anyways?

Reply:
done